src/app/zones/views/ZoneIngressListView.vue

```diff
@@ -13,142 +15,101 @@
         },
       ]"
     >
-      <div class="zoneingresses">
-        <MultizoneInfo v-if="store.getters['config/getMulticlusterStatus'] === false" />
+      <MultizoneInfo v-if="store.getters['config/getMulticlusterStatus'] === false" />
 
-        <!-- Zone CPs information for when Multicluster is enabled -->
-        <div
-          v-else
-          class="stack"
+      <template v-else>
+        <DataSource
+          v-slot="{ data: zoneIngressOverviewData, error }: ZoneIngressOverviewCollectionSource"
```

Comment:
Same comment here on the renaming, can we just stick with `data` without the rename?